README.md

```diff
@@ -231,6 +237,51 @@ By default, `beekeeper-scheduler-apiary` listens on port 8080, `beekeeper-path-c
 | `scheduler-delay-ms`                | No       | Amount of time (in milliseconds) between consecutive cleanups. Default value is `300000` (5 minutes after the previous cleanup completes). |
 | `Metastore-uri`                     | Yes      | URI of the Hive Metastore where tables to be cleaned-up are located. |
 
+## Beekeeper-API
```

Comment:
can you mention the swagger enpoint somewhere in here and how to reach it when beekeper api is deployed. It's a good starting point for most users to explore capabiities